src/data/fintype/basic.lean

```diff
@@ -87,12 +87,15 @@ class fintype (α : Type*) :=
 namespace finset
 variables [fintype α] {s : finset α}
 
+instance : order_top (finset α) :=
+{ top := fintype.elems α,
+  le_top := λ s x hx, fintype.complete x }
+
 /-- `univ` is the universal finite set of type `finset α` implied from
   the assumption `fintype α`. -/
-def univ : finset α := fintype.elems α
+def univ : finset α := ⊤
```

Comment:
I didn't get if this change is actually needed for something below, or is there another reason?

Reply:
I thought that this change will make order lemmas apply without `@` but I was wrong.

Reply:
I reverted this part of the diff.